test/test_core_misc.py

```diff
@@ -871,13 +871,13 @@ def test_pickle_categorical(self):
 
 class TestHyperparamRanges(unittest.TestCase):
     def exactly_relevant_properties(self, keys1, operator):
-        def sorted(ll):
+        def sorted_copy(ll):
```

Comment:
Python has a built-in function [sorted](https://docs.python.org/3/library/functions.html#sorted) that should already do the right thing, so we can just elide this definition altogether.

Reply:
even better: the check can be replaced with
`self.assertCountEqual(keys1, keys2)`
obviating the need for `sorted`